src/source/PeerConnection/Rtcp.c

```diff
@@ -2,31 +2,125 @@
 
 #include "../Include_i.h"
 
+STATUS findTransceiverBySsrc(PKvsPeerConnection pKvsPeerConnection, PKvsRtpTransceiver* ppTransceiver)
+{
+    STATUS retStatus                = STATUS_SUCCESS;
+    PDoubleListNode pCurNode        = NULL;
+    UINT64 item                     = 0;
+    PKvsRtpTransceiver pTransceiver = NULL;
+
+    CHK_STATUS(doubleListGetHeadNode(pKvsPeerConnection->pTransceievers, &pCurNode));
+    while (pCurNode != NULL) {
+        CHK_STATUS(doubleListGetNodeData(pCurNode, &item));
+        pTransceiver = (PKvsRtpTransceiver) item;
+        if (pTransceiver->sender.ssrc || pTransceiver->sender.rtxSsrc) {
+            break;
+        }
+    }
+    *ppTransceiver = pTransceiver;
```

Comment:
What if `pTransceiver` is NULL? Any repurcussions while making the call to this function?

Reply:
noted